### PR TITLE
f3d0t Html-css-popup HomeWork

### PR DESCRIPTION
**Pull request by f3d0t**
### Html css popup

[Demo](https://f3d0t.github.io/f3d0t/html_css_popup/)
[Code base](https://github.com/f3d0t/f3d0t/tree/main/html_css_popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Review by lenadgit — CHANGES_REQUESTED** [review]
Good job! remains only to format your code by autoformatting (html, css)

**Comment by f3d0t** [comment]
Formatting done

File: submissions/f3d0t/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+                </li>
+                <li class="menu__item">
+                    <a class="menu__item--link--img" href="#">
+                        <img alt="Bell button" class="popup__button--img" src="img/bell.png">
+                        </img>
+                    </a>
+                </li>
+                <li class="menu__item">
+                    <a class="menu__item--link--img" href="#">
+                        <img alt="User avatar" class="popup__button--img avatar" src="img/apps.png">
```
Comment (Temu4) [inline submissions/f3d0t/html-css-popup/index.html:173]: Can you play around icon images and decrease unnecessary sizes?
Screenshot 2020-12-11 at 19 07 19

**Review by Temu4 — CHANGES_REQUESTED** [review]
Well done, just one more thing ⬇️

**Comment by f3d0t** [comment]
Pics size decreasing done

**Comment by Temu4** [comment]
> Pics size decreasing done

Screenshot 2020-12-11 at 23 07 52

**Comment by f3d0t** [comment]
images done =)

**Comment by Temu4** [comment]
Looks good to me! Well done! 👍

**Pull request merged**